tests/testthat/test-epinowcast.R

```diff
@@ -62,9 +68,7 @@ test_that("epinowcast() runs using default arguments only", {
 test_that("epinowcast() runs with within-chain parallelisation", {
   skip_on_cran()
   skip_on_local()
-  obs <- germany_covid19_hosp[age_group %in% "00+"][location %in% "DE"] |>
-    enw_filter_report_dates(remove_days = 10) |>
-    enw_filter_reference_dates(include_days = 10)
+  obs <- run_window_filter(germany_covid19_hosp[age_group %in% "00+"][location %in% "DE"])
```

Comment:
hmm did we not get all the `%in%` => `==`s?

Reply:
Replaced these `%in%`s with `==` is commit [48c3a38](https://github.com/epinowcast/epinowcast/pull/402/commits/48c3a3850c2e715200f3b4633a2d161bc8baebcb)